LEGO1/mxnotificationmanager.cpp

```diff
@@ -1,15 +1,145 @@
+#include "legoomni.h"
+#include "mxautolocker.h"
+#include "mxcore.h"
 #include "mxnotificationmanager.h"
+#include "mxparam.h"
+#include "mxtypes.h"
 
-// OFFSET: LEGO1 0x100ac450 STUB
+#include "compat.h"
+#include "decomp.h"
+
+DECOMP_SIZE_ASSERT(MxNotification, 0x8);
+// Can't use DECOMP_SIZE_ASSERT due to STL type size changes.
+//DECOMP_SIZE_ASSERT(MxNotificationManager, 0x40);
```

Comment:
I think we can keep this assert because `DECOMP_SIZE_ASSERT` will probably get nopped on newer compilers.

Reply:
OK. Maybe we can just nop if `_MSC_VER > 1020`? The concept of "accuracy" doesn't really apply on different compilers, and I'm not aware of any instances where we depend on type size for correctness yet.

Reply:
Yes I think we can pretty safely assume any compiler that isn't MSVC 4.2 has no expectation of instruction accuracy.